### ISSUE TITLE
SocialPredict Script To-Do

### ISSUE THREAD

**Issue by ntoufoudis** [openpredictionmarkets/socialpredict#215]
@pwdel 
I created the file [SocialPredict Script To-Do.md](https://github.com/openpredictionmarkets/socialpredict/blob/deploy_prod/README/SocialPredict%20Script%20To-Do.md) and added some features that I would like to add to the script.

Please review it and let me know your thoughts.

Also feel free to add/remove/edit as you wish.

Finally if you can, sort them in the order you wish them completed.

**Comment by pwdel** [comment]
Updated.

Basically, the main thing in my opinion right now is to ensure that the passing criteria here:

https://github.com/openpredictionmarkets/socialpredict/pull/202#issue-2403781445

Gets passed, so that we can mine for more user opinions. I think prioritization will be much easier if we get actual user complaints. I think we should, "crave" complaints because they will help us understand end user reality a bit more rather than working on things that are based upon a mental model.

Or rather, we can improve our mental models based upon user feedback.

So basically, I divided that into, "Essential" whereas everything else is, "Nice to Have," and I think once we have that essential task completed, we can re-order, "Nice to Have" over time as new information comes in.

It would be good to try to get @astrosnat and @jg-you and perhaps @joy-void-joy (if they would like) to spin up a prod instance of their own and give us feedback on what the biggest problems were.

**Comment by astrosnat** [comment]
I'll try to spin up 2 prod instances tomorrow (1 on Linux, 1 on Windows/WSL2) and report back

**Comment by pwdel** [comment]
Greetings @astrosnat , what service are you going to use for prod? When I hear, "prod instances," I think, Digital Ocean / Ubuntu. Typically windows is not used for prod.

Did you mean, use prod on local? If so, I am not sure what the value of this would be.

Deploying `dev` to local, meaning, pull this branch here specifically: https://github.com/openpredictionmarkets/socialpredict/tree/deploy_prod  ...and then running that on local with `./SocialPredict install` and selecting "development" and then running `./SocialPredict up` ... would be helpful.

After running those steps, you can check `localhost` in your browser (no port).

This is going to be the new workflow for deploying to both `dev` on local and `prod` on services like Digital Ocean.

**Comment by astrosnat** [comment]
You're correct, I just hadn't had enough coffee to write correctly. Thank you!

**Comment by pwdel** [comment]
It happens!
